client/js/widgets/button.js

```diff
@@ -133,7 +148,10 @@ class Button extends Widget {
         setDefaults(a, { value: 0, mode: 'set' });
         if([ 'set', 'dec', 'inc' ].indexOf(a.mode) == -1)
           problems.push(`Warning: Mode ${a.mode} will be interpreted as add.`);
-        this.w(a.label, label=>label.setText(a.value, a.mode));
+        this.w(a.label, label=> {
+          if (this.p('debug')) console.log(`changing ${a.label} ${a.mode} ${a.value}`)
```

Comment:
I'm guessing this is left over debug code? Or do you react to these with the test framework? I guess I'll find out. :D

Reply:
I used it throughout development of the test, but then disabled usage of debug before committing my changes. For the purpose of achieving higher coverage, we should creating an internal log function that is always called and have it check if debug is on.

Reply:
I don't really care. I would just like the console to be clean if no button is being debugged.